README.md

```diff
@@ -1,23 +1,78 @@
+# Generic Torquer – NOS3 Component
+This repository contains the NOS3 Generic Torquer Component. This includes flight software (fsw), ground software (gsw), and simulation (sim) directories.
 
-# Generic_torquer - NOS3 cFS Application
+## Overview
+Torquers, or magnetorquers, are commonly used on spacecraft as a secondary attitude control mechanism. They consist of electromagnets placed asymmetrically. Different currents running through them then generate different magnetic fields, and the torque created by the interaction of the spacecraft magnetic field and the Earth’s magnetic field rotates the vehicle. Because of the small amounts of torque available, magnetorquers are only infrequently used as the primary attitude control mechanism; they frequently serve as a secondary mechanism, though, and are used to provide small rotational changes or to gradually unload reaction wheels. 
+The generic torquer is a device which accepts as input a current (as a percent of the maximum) and returns a value of the generated magnetic field. 
+The available flight software is for use in the core Flight System (cFS) while the ground software supports COSMOS.
+A NOS3 simulation is available which uses a 42 data provider.
 
-This repository contains the NOS3 Generic_torquer cFS application.
+## Technical Specifications
+- Maximum magnetic field strength __
```

Comment:
Can remove this `Technical Specifications` section since those options are captured later